guides/V1_wwsg_Webhooks.md

```diff
@@ -33,6 +33,8 @@ Available events are:
 
 **message-deleted**: each time a message is deleted from the space. Please note that the **message-deleted** event API is currently `experimental` and subject to change. It is being included here to engage developers early with this API.
 
+**message-edited**: each time message content is edited in the space.  Please note that the triggering of message-edited events takes place due to a change in the data model via Administrative, End User or Programmatic means.  Currently only Administrative means are implemented.
```

Comment:
_these sentences seems overly cryptic_

> Please note that the triggering of message-edited events takes place due to a change in the data model via Administrative, End User or Programmatic means.  Currently only Administrative means are implemented.

_how about_

Please note that the triggering of message-edited events takes place due to a change in the data model via Administrator action, End User action or through an API call.  Currently only Administrator actions  are implemented.